client/src/components/settings/privacy.tsx

```diff
@@ -22,7 +22,7 @@ const mapStateToProps = createSelector(userSelector, user => ({
 const mapDispatchToProps = (dispatch: Dispatch) =>
   bindActionCreators({ submitProfileUI }, dispatch);
 
-type ProfileUIType = {
+type ProfileUI = {
```

Comment:
All of these should probably be added to the `User.profileUI` def in `prop-types.ts`